kamelet-support/src/test/java/io/kaoto/backend/metadata/parser/step/kamelet/KameletParseCatalogTest.java

```diff
@@ -10,21 +12,20 @@
 import io.quarkus.test.kubernetes.client.WithKubernetesTestServer;
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.Timeout;
 
 import javax.inject.Inject;
+import java.io.IOException;
+import java.net.MalformedURLException;
 import java.net.URISyntaxException;
+import java.net.URL;
 import java.nio.file.Path;
 import java.util.Arrays;
 import java.util.Comparator;
 import java.util.List;
 import java.util.Objects;
-import java.util.concurrent.TimeUnit;
 import java.util.stream.Collectors;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
We try not to compress the imports, for clarity.

Reply:
ok, I will revert it. Also, I can extend main [checkstyle](https://github.com/KaotoIO/kaoto-backend/blob/main/checkstyle.xml) file about it.